tileview/src/main/java/com/qozix/tileview/TileView.java

```diff
@@ -772,7 +772,9 @@ public void onPanEnd( int x, int y, Origination origin ) {
 
   @Override
   public void onZoomBegin( float scale, Origination origin ) {
-    mDetailLevelManager.lockDetailLevel();
+    if( !Origination.DOUBLE_TAP.equals( origin ) ) {
```

Comment:
My bad i wasn't clear, i wanted to say "why should we forbid detail level change".